Dockerfile

```diff
@@ -1,5 +1,4 @@
 FROM python:2.7-slim
-ENV PYTHONUNBUFFERED 1
```

Comment:
Still needed if somebody needs to up container manually for debug.

Reply:
👍 That's reasonable!